sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImpl.java

```diff
@@ -141,23 +159,31 @@ public SerializerAdapter getSerializerAdapter() {
      * The interface defining all the services for TextAnalyticsClient to be used by the proxy service to perform REST
      * calls.
      */
-    @Host("{Endpoint}/text/analytics/v3.1-preview.4")
+    @Host("{Endpoint}/text/analytics/{ApiVersion}")
```

Comment:
with this change(autogene), we are able to request with a different service API version